.buildkite/pipeline.macos.yml

```diff
@@ -110,20 +110,6 @@ steps:
     - if [ "$BUILDKITE_BRANCH" = "master" ]; then python .buildkite/copy_files.py --destination wheels --path ./.whl; fi
     - if [ "$BUILDKITE_BRANCH" = "master" ]; then python .buildkite/copy_files.py --destination jars --path ./.jar/darwin; fi
 
-
-- label: ":mac: :apple: Ray Core Unit Tests and Dashboard"
-  <<: *common
-  conditions: ["RAY_CI_CORE_CPP_AFFECTED", "RAY_CI_PYTHON_AFFECTED", "RAY_CI_DASHBOARD_AFFECTED"]
-  commands:
-    - *prelude_commands
-    - TORCH_VERSION=1.9.0 ./ci/env/install-dependencies.sh
-    # Use --dynamic_mode=off until MacOS CI runs on Big Sur or newer. Otherwise there are problems with running tests
-    # with dynamic linking.
-    - bazel test --config=ci --dynamic_mode=off --test_env=CI $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=-post_wheel_build --
-      //:all python/ray/dashboard/... -python/ray/serve/... -rllib/... -core_worker_test
-    - *epilogue_commands
-
-
 - label: ":mac: :apple: Ray C++ and Java"
```

Comment:
this test is still here? not entirely "the rest of mac tests"? :)

Reply:
ah yes not everything yet